lib/spack/spack/test/cmd/install.py

```diff
@@ -913,6 +919,7 @@ def test_cdash_auth_token(tmpdir, mock_fetch, install_mockery, capfd):
             assert "Using CDash auth token from environment" in out
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Logger race condition in windows")
```

Comment:
Is this a unique error compared to the other notes here about calling `log_output` in `install()`? I ask because the error message differs

Reply:
This is the same reason as the others. I can switch this one to match the others.